mobileweb/titanium/Ti/UI/Animation.js

```diff
@@ -423,6 +423,7 @@ define(['Ti/_/declare', 'Ti/_/Evented', 'Ti/_/style', 'Ti/UI'], function(declare
 
 		promise.pause = function() {
 			var a = findAnimation();
+			updateRect(a.elem);
```

Comment:
I do recall there being some reason for this. However, I made this modification over a year ago, so I'm not quite sure what the reason was. I know at least one reason was that KineticScroll views are completely broken without this fix. Stopping a scrollview mid scroll, lead to that scroll view reverting to it's original position.

Everytime I look at Animation.js I want to start from scratch, letting the browser (and thus the GPU) do all the heavy lifting. It seems like we could eliminate more than 50% of the code and get a huge performance boost by using CSS-based animation rather than tweening the transform property.

Does Appcelerator by chance maintain any kind of documentation regarding the inner workings of mobile web?